src/libraries/System.Security.Cryptography.Encoding/tests/Cbor/CborInitialByte.cs

```diff
@@ -18,36 +21,37 @@ internal enum CborMajorType : byte
         Simple = 7,
     }
 
+    /// <summary>
+    ///   Represents the 5-bit additional information included in a CBOR initial byte.
+    /// </summary>
     internal enum CborAdditionalInfo : byte
     {
-        SimpleValueFalse = 20,
-        SimpleValueTrue = 21,
-        SimpleValueNull = 22,
-
         Additional8BitData = 24,
         Additional16BitData = 25,
         Additional32BitData = 26,
         Additional64BitData = 27,
         IndefiniteLength = 31,
     }
 
-    /// Represents the Cbor Data item initial byte structure
+    /// <summary>
+    ///   Represents a CBOR initial byte
+    /// </summary>
     internal readonly struct CborInitialByte
     {
         public const byte IndefiniteLengthBreakByte = 0xff;
-        private const byte AdditionalInformationMask = 0b000_11111;
+        public const byte AdditionalInformationMask = 0b000_11111;
 
         public byte InitialByte { get; }
 
-        internal CborInitialByte(CborMajorType majorType, CborAdditionalInfo additionalInfo)
+        public CborInitialByte(CborMajorType majorType, CborAdditionalInfo additionalInfo)
         {
             Debug.Assert((byte)majorType < 8, "CBOR Major Type is out of range");
             Debug.Assert((byte)additionalInfo < 32, "CBOR initial byte additional info is out of range");
```

Comment:
If this is supposed to be a public ctor the asserts need to be replaced with runtime checks and throws.

Reply:
It's an internal type (and will stay that way). I changed the particular ctor to public for consistency with the other public members in the type.